src/App.js

```diff
@@ -36,19 +39,21 @@ const donations = [
   },
 ];
 
-function App() {
-  return (
-    <>
-      <TopBar />
-      <main className="container">
-        <section className="sidebar">{/* Recent Donations */}</section>
-        <section className="">
-          {/* Progress */}
-          {/* Donation Form */}
-        </section>
-      </main>
-    </>
-  );
+export default class App extends React.Component {
+  render() {
+    return (
+      <>
+        <TopBar />
+        <main className="container">
+          <section className="sidebar">
+            <RecentDonations donations={donations}/>
+          </section>
+          <section className="">
+            <Progress max={targetAmount}/>
```

Comment:
Aside from the target amount, you also need the sum of the donations already made. What variable holds that data? you would need to pass that as a prop as well.